guides/configuration.md

```diff
@@ -0,0 +1,102 @@
+  config.use_static_preferences!
+
+  config.currency = "USD"
+  config.mails_from = "store@example.com"
+end
+```
+
+This block instantiates the main configuration object for the Solidus engine. Here, you can change Solidus' behaviour to accomodate many common use cases, like using another default currency or another sender e-mail address.
+
+If your desired change in behaviour can be accomplished with one of the configuration options available here (see the full list in [our YARD docs for the `Spree::AppConfiguration` class](http://docs.solidus.io/Spree/AppConfiguration.html)), you should do this. These are tested options that allow you to implement complex behaviour with little effort and well-tested code.
+
+## Overriding templates
+
+### In the frontend
+
+If you want to change something about HTML structure of Solidus' frontend, the customer-facing part of Solidus, your best bet is copying the views from Solidus' frontend gem and adding them to your host app in the exact same location.
+
+For example, if you want to change the way the product list is rendered, take the raw version of [that template] (https://github.com/solidusio/solidus/blob/master/frontend/app/views/spree/products/index.html.erb) and copy it to `app/views/spree/products/index.html.erb`. It is very important that the path and name of this file are exactly the same as in the Gem! Template files from your app will always win over template files from an imported engine.
+
+Of course, it the template you wish to modify references other template files, you can override those, too. For example, the product list references a shared partial `app/views/spree/shared/_products.html.erb` - you can find it [here](https://github.com/solidusio/solidus/blob/master/frontend/app/views/spree/shared/_products.html.erb) and also override it in your host app.
+
+When you override a view from a Gem, there is the possibility of difficulties when upgrading, because Controller code within the Gem might have changed, such as for example the name of an instance variable. Be careful when upgrading, and always read `CHANGELOG.md`.
+
+### In the backend
+
+We do not recommend overriding the admin backend in the same way, since some of the extensions you might be running (see below) depend on them having the HTML structure present in the version of Solidus they were built for. If you need to add custom functionality to Solidus' backend, we recommend using the `Deface` gem. Find it [here](https://github.com/spree/deface). Overriding backend views with Deface is tricky - be sure to have good tests!
+
+## Changing assets
+
+If you need to change the look and feel of your app without changing the HTML structure, you can customize the CSS, image and JavaScript assets by amending the files in `vendor/assets/*/*/all.js`. They mirror the structure of the `app/assets` folder. Have a look at the [Spree Guide](http://guides.spreecommerce.org/developer/asset.html) for overriding assets, it applies to Solidus as-is.
```

Comment:
I think it's kinda dangerous to link to the spree guides. We can't rely on them being online forever.